components/select-control/index.js

```diff
@@ -10,9 +10,25 @@ import BaseControl from '../base-control';
 import withInstanceId from '../higher-order/with-instance-id';
 import './style.scss';
 
-function SelectControl( { label, help, instanceId, onChange, options = [], ...props } ) {
+function SelectControl( {
+	help,
+	instanceId,
+	label,
+	multiple = false,
+	onChange,
+	options = [],
+	...props
+} ) {
 	const id = `inspector-select-control-${ instanceId }`;
-	const onChangeValue = ( event ) => onChange( event.target.value );
+	const onChangeValue = ( event ) => {
+		if ( multiple ) {
+			const selectedOptions = [ ...event.target.options ].filter( ( { selected } ) => selected );
```

Comment:
Lodash can be helpful here, both for picking on property, and for casting to array:

```js
const selectedOptions = filter( event.target.options, 'selected' );
const newValues = map( selectedOptions, 'value' );
```